src/FixBookUI.java

```diff
@@ -47,7 +47,7 @@ public void run() {
                 break;
 
             case FIXING:
-                String AnS = input("Fix Book? (Y/N) : ");
+                String AnS = askQuestion("Fix Book? (Y/N) : ");
                 boolean FiX = false;
                 if (AnS.toUpperCase().equals("Y")) {

```

Comment:
should askQuestion cover both upper and lowercases? if the user enters lower case y/n, does it bomb out? If so, perhaps need to put in place that if lowercase, then change case to upper?

Reply:
.toUpperCase() is there to do that.